### PR TITLE
cover page created .

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes: #2 

you can view the cover page : https://youthful-liskov-ec4953.netlify.app/

### CODE REVIEW

**Comment by nlok5923** [comment]
@AmanRaj1608 sir the cover page i has created is not responsive so sir for it. Will you make another issue to create cover page responsive or in this pr only i has to make it responsive.

**Comment by nlok5923** [comment]
@AmanRaj1608 sir please review this pull request.

File: src/components/Home/Cover/Cover.css [hunk 1]
```diff
@@ -1,63 +1,66 @@
...
+  letter-spacing: -0.05em;
+  color: #ffffff;
+}
+.inf-btn {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 65px;
+}
+.trans-btn {
```
Comment (AmanRaj1608) [inline src/components/Home/Cover/Cover.css:47]: Can we have some hover effect.

**Review by AmanRaj1608 — CHANGES_REQUESTED** [review]

**Comment by AmanRaj1608** [comment]
Hi, @nlok5923 sorry for the late review,
It's okay I will create another issue for responsive design give it a try if you can fix it.

**Comment by nlok5923** [comment]
@AmanRaj1608 sir added the hover effect feature.

**Review by AmanRaj1608 — APPROVED** [review]

**Pull request merged**